client/src/components/Create.jsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Box,
+  TextField,
+  Stack,
+  Button,
+  Card,
+  CardHeader,
+  CardContent,
+  Typography,
+} from "@mui/material";
+
+const styles = {
+  body: {
+    height: "100vh",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+    fontFamily: "Helvetica",
+    backgroundColor: "#f0f3f8",
+  },
+  heading: {
+    display: "flex",
+    width: "47.5rem",
+    paddingRight: "0px",
+    flexDirection: "column",
+    alignItems: "flex-start",
+    gap: "0.625rem",
+  },
+  cancel: {
+    marginRight: "auto",
+  },
+};
+
```

Comment:
We should add a function for the Button onClick and a fake API call so when the backend gets implemented, it will be a quick edit and test to get it hooked up.

Reply:
Also you should probably maintain the states of the different fields in this component so that we can track what the user has inputted and attach that to the API call. This also applies in edit.jsx